app/views/layouts/application.html.haml

```diff
@@ -1,58 +1,70 @@
 !!! 5
+<!DOCTYPE html>
+<!--[if IE]><![endif]-->
```

Comment:
It begins the IE conditional comment:

<img width="637" alt="screen shot 2016-08-08 at 4 31 48 pm" src="https://cloud.githubusercontent.com/assets/81432/17471186/afb40044-5d85-11e6-906f-4ab1d6fdf0b1.png">